recipes/default.rb

```diff
@@ -15,3 +15,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# include_recipe php
+# include_recipe mysql
+
+# Following this tutorial: https://linuxize.com/post/set-up-an-email-server-with-postfixadmin/
+
+group 'vmail' do
+  gid 5000
+end  
+
+user 'vmail' do
+  comment 'Owner of all mailboxes. Used to access emails on this server'
+  home    '/var/mail/vmail'
+  uid     5000
+  gid     5000
+  shell   '/usr/sbin/nologin'
+end
+
+# For testing
+
+user 'www-data' do
+  comment 'Php user'
+  uid     5001
+  gid     5000
+end
+
+directory '/var/www'
```

Comment:
FWIW when you include osl-apache, this will be automatically done so you might just replace this with that.